icone_arrow_boards/final_icone_report.py

```diff
@@ -53,8 +59,12 @@ def check_id(id):
 
 
 INITIAL_PATH = "./icone_arrow_boards/2023_05_16/"
```

Comment:
Is this script only ever used for the single day report?

Reply:
The script works from any folder, so simple 'icone_arrow_boards" would run all of the collected data. I was re-running some individual days when I checked this in, so it's in this state.